### PR TITLE
Bake 1.2

### PR DESCRIPTION
**Pull request by lorenzo**
This is to be merged right before cake 3.2 is marked as stable

### CODE REVIEW

**Comment by markstory** [comment]
Looks good. I updated the travis config.

**Comment by josegonzalez** [comment]
The comparison files still need to be updated.

**Comment by dereuromark** [comment]
Looks like this should be merged now for the 3.2 version tagged.

**Comment by markstory** [comment]
@dereuromark I'll take care of that.

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": ">=5.4",
```
Comment (ADmad) [inline composer.json]: This should be bumped to 5.5 isn't it?

Comment (markstory) [inline composer.json]: I can do that.

Comment (ADmad) [inline composer.json]: Made a PR fixing this and few other things

**Pull request merged**